app/src/test/java/org/mozilla/fenix/settings/logins/biometric/BiometricPromptFeatureTest.kt

```diff
@@ -0,0 +1,105 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.settings.logins.biometric
+
+import android.os.Build.VERSION_CODES.M
+import android.os.Build.VERSION_CODES.N
+import androidx.biometric.BiometricManager.Authenticators.BIOMETRIC_WEAK
+import androidx.biometric.BiometricManager.Authenticators.DEVICE_CREDENTIAL
+import androidx.biometric.BiometricPrompt
+import androidx.fragment.app.Fragment
+import io.mockk.mockk
+import io.mockk.slot
+import io.mockk.verify
+import junit.framework.Assert.assertFalse
+import mozilla.components.support.test.robolectric.createAddedTestFragment
+import mozilla.components.support.test.robolectric.testContext
+import org.junit.Assert.assertEquals
+import org.junit.Assert.assertNotNull
+import org.junit.Assert.assertNull
+import org.junit.Assert.assertTrue
+import org.junit.Before
+import org.junit.Ignore
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.mozilla.fenix.helpers.FenixRobolectricTestRunner
+import org.robolectric.annotation.Config
+
+@RunWith(FenixRobolectricTestRunner::class)
+class BiometricPromptFeatureTest {
+
+    lateinit var fragment: Fragment
+
+    @Before
+    fun setup() {
+        fragment = createAddedTestFragment { Fragment() }
+    }
+
+    @Config(sdk = [N])
+    @Test
+    fun `canUseFeature checks for SDK compatible`() {
+        assertFalse(BiometricPromptFeature.canUseFeature(testContext))
+    }
+
+    @Config(sdk = [M])
+    @Test
+    @Ignore("Figure out how to mock the BiometricManager.from(context) call.")
```

Comment:
We can use static mocks (similar to these examples https://mockk.io/#extension-functions)
and setup this test with something like
>        mockkStatic("org.mozilla.fenix.settings.logins.biometric.ext.BiometricManagerKt")
>        val manager: BiometricManager = mockk()
>        every { manager.isHardwareAvailable() } returns true
>        every { manager.isEnrolled() } returns false
>        mockkStatic(BiometricManager::class)
>        every { BiometricManager.from(any()) } returns manager

`<... actual verifications>`

followed at the end of the test by calls to `unmockkStatic(..)`

Reply:
Thanks! I went with just mocking out the `BiometricManager.canAuthenticate` since the extensions were simple condition checks.